assets/css/_diversion_page.scss

```diff
@@ -107,3 +109,7 @@
 .c-diversion-panel__help-text {
   line-height: 1.5;
 }
+
+.c-diversion-panel__activate-button {
+  background-color: semantic.$missed-stop;
+}
```

Comment:
Do we need bespoke css for this? I'd prefer to use a `variant` on the button itself https://react-bootstrap.github.io/docs/components/buttons

Reply:
Ahh - I didn't realize I could do [that](https://github.com/mbta/skate/pull/2713/commits/a9c43b6ba83f5d10c7e69f2268f8565334c65619)

(It gives it nicer hover behavior too!)